actions/plugins/publish/change-plugin-scope/change-plugin-scope.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
```

Comment:
@oshirohugo might be too late to suggest this so if you don't feel like re-doing the work it's ok, but I always find these bash scripts harder to follow. 

Consider maybe migrating this script to nodejs and simply adding a setup-node step before this one.